troubleshooting/app-does-not-connect-to-remote-point.md

```diff
@@ -0,0 +1,53 @@
+tags: connect, ios, web-service
+ticketid: 1061272, 1029681, 1011781, 1006104
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Run on Device</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+</table>
+
+## Description
+When running an app on a device, its requests to a remote point via HTTPS do not work, even though the app works in the device simulator. Upon expecting the error arguments of the failed request, you find `statusText: error` and no error message or response text.
+
+The issue is observed on iOS devices with any build configuration and on Android devices when the app is built with **Release** configuration.
+
+## Cause
+This problem occurs when the remote server uses an untrusted SSL certificate. This could happen if the server uses a certificate form a trusted authority but does not have any intermediate certificates installed, or if the server uses a self-signed certificate. You can check if there is such a problem with your server certificate by running it through an online SSL certificate checker tool like [Symantec CryptoReport](https://cryptoreport.symantec.com/checker/).
+
+Android devices allow connections via HTTPS to a server using an untrusted certificate only when you build the app with **Debug** build configuration, to let you test your app. When the app is built with **Release** configuration, connections to such servers are blocked.
+
+iOS devices block any HTTPS requests to untrusted remote points, no matter what build configuration was used to produce the app package.
+
+## Solution
+Use a trusted certificate on the server that you want to connect to. If you are already using a certificate issued by a trusted certificate authority, you need to check for any missing intermediate certificates that should link your certificate to the root certificate of the issuer. 
+
+## Suggested Workaround
+Sometimes, using a trusted certificate is not possible for development and testing scenarios, or you may need a quick fix until the server is correctly configured. In these cases, you can choose to modify the behavior of the iOS app and prevent it from blocking the request. You can achieve this using the third-party [Certificate Plugin](https://github.com/hypery2k/cordova-certificate-plugin). To do it follow these steps:
+
+1. [Import the plugin in your app](http://docs.telerik.com/platform/appbuilder/cordova/using-plugins/using-custom-plugins/add-custom-plugins).
+2. Open your app source code and copy this line to the `deviceready` event handler:
+	
+	```	
+	cordova.plugins.certificates.trustUnsecureCerts(true);
+	```
+3. Build and deploy your app on a device.
+
+## Notes
+The suggested workaround is only applicable during the development stage of your app. When you publish your app, you need to remove the custom plugin and ensure that the remote server uses a trusted certificate.
```

Comment:
The note about development purposes should be in the workaround section, imo. The notes should indicate that you should always ensure your server is using a trusted certificate and you are using secure connection.